app/views/insured/families/_consumer_brokers_widget.html.erb

```diff
@@ -2,6 +2,7 @@
   <% current_broker_agency = @person&.primary_family&.current_broker_agency %>
   <% current_assister_agency = @person&.primary_family&.current_assister_agency %>
   <% writing_agent =  @person&.primary_family&.current_broker_agency&.writing_agent %>
+  <% genesys_feature_enabled = EnrollRegistry.feature_enabled?(:genesys_cloud_messenger) %>
```

Comment:
Not a huge deal, but with the number of times we're adding this check might be worth setting up a helper method for it rather than redeclaring it every time